support-workers/cloud-formation/src/templates/lambda.yaml

```diff
@@ -9,8 +9,25 @@
     Role: !GetAtt [ LambdaExecutionRole, Arn ]
     Code:
       S3Bucket: support-workers-dist
-      S3Key: !Sub support/${Stage}/support-workers/support-workers.jar
+      S3Key: !Sub support/${Stage}/support-workers/support-workers-{{> GITHUB_RUN_NUMBER}}.jar
+    SnapStart:
+      ApplyOn: PublishedVersions
     Runtime: "java21"
     Timeout: "60"
     Architectures:
-      - arm64
+      - x86_64
+
+{{name}}AliasForLambda:
+  Type: "AWS::Lambda::Alias"
+  Properties:
+    FunctionName: !Ref {{name}}Lambda
+    FunctionVersion: !GetAtt [ {{name}}CurrentVersion, Version ]
+    Name: SNAPSTART
```

Comment:
wasn't sure what to call the alias, it's just the latest production version, but I don't want to confuse with `$LATEST` by calling it `latest` or similar.